### PR TITLE
Add challenge page

### LINKED ISSUE #2: Implement challenge page
**Short description**
We need a page with the challenge 

**Acceptance criteria**
should be accessed through the navigation bar
Should have a title of the challenge
Should have a description of the challenge
Should have a bullet list of the rules of the challenge

### PR DESCRIPTION
**Pull request by OleVedeler**
fixes #2

### CODE REVIEW

File: pages/challenge.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import Head from "next/head"
+
+export default function Web() {
+  return (
+    <>
+      <Head>
+        <meta property="og:url" content="https://next-enterprise.vercel.app/" />
+        <meta
+          property="og:image"
+          content="https://raw.githubusercontent.com/Blazity/next-enterprise/main/project-logo.png"
```
Comment (Benjamsk) [inline pages/challenge.tsx]: Can we please remove these meta-tags? These doesnt make sens for us

File: pages/challenge.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        />
+        <meta property="og:image:width" content="1200" />
+        <meta property="og:image:height" content="630" />
+        <meta name="twitter:card" content="summary_large_image" />
```
Comment (Benjamsk) [inline pages/challenge.tsx]: And these

File: pages/challenge.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        <title>Oles eleven</title>
+      </Head>
+      <section className="bg-white dark:bg-gray-900">
+        <div className="mx-auto grid max-w-screen-xl px-4 py-8 text-center lg:py-16">
+          <div className="mx-auto place-self-center">
+            <h1 className="mb-4 max-w-2xl text-4xl font-extrabold leading-none tracking-tight dark:text-white md:text-5xl xl:text-6xl">
+              11 Miles challenge
```
Comment (Benjamsk) [inline pages/challenge.tsx]: Would rename to "11 Scandinavian Miles"
And drop the word challenge

File: pages/challenge.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            </h1>
+            <p className="mb-6 max-w-2xl font-light text-gray-500 dark:text-gray-400 md:text-lg lg:mb-8 lg:text-xl">
+              This challenge is to complete 11 miles (110 kilometers) in under 24 hours by walking and running. It must
+              include atleast one hour of running. it can not include any othe form of transportation.
```
Comment (Benjamsk) [inline pages/challenge.tsx]: Suggest changing text to this:

The 11 Scandinavian Miles is the challenge to traverse 110 kilometers by walking and running within 24 hours. The two rules are simple: 1) no other mode of transportation than your own legs, and 2) at least 10 kilometers of the distance must be traversed while running.

**Review by Benjamsk — APPROVED** [review]

File: pages/challenge.tsx (hunk 1 above)
Comment (OleVedeler) [reply pages/challenge.tsx]: done

File: pages/challenge.tsx (hunk 2 above)
Comment (OleVedeler) [reply pages/challenge.tsx]: done

File: pages/challenge.tsx (hunk 3 above)
Comment (OleVedeler) [reply pages/challenge.tsx]: done

File: pages/challenge.tsx (hunk 4 above)
Comment (OleVedeler) [reply pages/challenge.tsx]: done

**Pull request merged**